IdentityCore/src/broker_operation/request/browser_native_message_request/MSIDBrokerOperationBrowserNativeMessageRequest.m

```diff
@@ -94,6 +132,10 @@ - (NSDictionary *)jsonDictionary
     
     json[MSID_BROWSER_NATIVE_MESSAGE_REQUEST_PAYLOAD_KEY] = [self.payloadJson msidJSONSerializeWithContext:nil];
     
+    json[MSID_BROWSER_NATIVE_MESSAGE_REQUEST_PPTID_KEY] = self.parentProcessTeamId;
```

Comment:
Are those guranteed to be non-null?

Reply:
MSID_BROWSER_NATIVE_MESSAGE_REQUEST_PPTID_KEY is constant, self.parentProcessTeamId can be nill

Reply:
I think it will crash if assign a `nil` value to a key in dictionary